source/_integrations/ibeacon.markdown

```diff
@@ -41,6 +41,10 @@ Consider setting up your iBeacons with a schema similar to the following:
 
 iBeacon devices that do not have stable Major and Minor values are not supported. The system automatically removes iBeacon devices with unstable Major and Minor values once ten (10) or more Major and Minor values have been seen with the same UUID from an iBeacon device with a fixed MAC address.
 
+Moreover, iBeacon devices that do not broadcast their device name are ignored (to avoid flooding your system with devices that are not real beacons).
+You can explicitly allow a list of UUIDs to be detected even with an empty device name in the integration options
+({% my integrations title="**Settings** > **Devices & Services**" %} > **iBeacon Tracker** > **Configure**).
```

Comment:
```suggestion
The system will not automatically create iBeacon devices for devices that do not broadcast their name to avoid flooding your system with devices that are not actual beacons.

To explicitly allow a list of UUIDs to be added even with an empty device name, add them via the integration options:

{% include integrations/option_flow.md %}
```